force-app/main/default/lwc/geFormWidgetSoftCredit/__tests__/geFormWidgetSoftCredit.test.js

```diff
@@ -1,17 +1,21 @@
 import { createElement } from 'lwc';
 import GeFormWidgetSoftCredit from 'c/geFormWidgetSoftCredit';
 
-const DUMMY_SOFT_CREDITS = [
+let DUMMY_SOFT_CREDITS = [
     { Role: 'Influencer', ContactId: '1', key: '1' },
     { Role: 'Honoree', ContactId: '2', key: '2' },
     { Role: 'Solicitor', ContactId: '3', key: '3' }
 ];
-const DUMMY_PROCESSED_SOFT_CREDITS = [
+let DUMMY_PROCESSED_SOFT_CREDITS = [
     { Role: 'Donor', ContactId: '4', Id: '1' },
     { Role: 'Household Member', ContactId: '5', Id: '2' },
 ];
 
 describe('c-form-widget-soft-credit', () => {
+
+    DUMMY_SOFT_CREDITS = JSON.stringify(DUMMY_SOFT_CREDITS);
+    DUMMY_PROCESSED_SOFT_CREDITS = JSON.stringify(DUMMY_PROCESSED_SOFT_CREDITS);
```

Comment:
I think we should probably avoid re-assignment here, we can rename either the original variable or the serialized one.